cmd/dmsg-server/commands/root.go

```diff
@@ -27,7 +27,7 @@ var (
 	cfgFromStdin bool
 )
 
-// Config is a messaging-server config
+// Config is a DMSG server config
```

Comment:
@evanlinjin I remember you said that lowercase spelling would be correct (`dmsg`). Is `DMSG` appropriate as well?

Reply:
@nkryuchkov I've always stuck with all lowercase. To be honest, I'm not sure what is best haha. What convention do you think we should have?

Reply:
@evanlinjin @Darkren Maybe `DMsg`?

Reply:
@nkryuchkov technically correct. But someone needs to go in and change it all haha.

Reply:
@evanlinjin @nkryuchkov should I change it? and what for? I didn't get the resume lol

Reply:
@Darkren @nkryuchkov Let's stick with lowercase `dmsg` for now. As that's how it is written everywhere else. We can talk about changes in the future.

Reply:
Done this